package.json

```diff
@@ -81,6 +81,7 @@
     "@vitest/coverage-v8": "^0.32.2",
     "@vitest/spy": "^0.32.2",
     "@vitest/ui": "^0.32.2",
+    "ajv": "^8.12.0",
```

Comment:
Remove?

Reply:
I think we should keep it. I'm using `ajv` in two places:

- `.vite/jsonSchemaPlugin.ts` (so `ajv` needs to be in `devDependencies` in `package.json`)
- `packages/mermaid/scripts/create-types-from-json-schema.mts` (so `ajv` should probably also be in `devDependencies` in `packages/mermaid/package.json`)